src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -263,7 +263,17 @@ export const WalletOptions: Definitions<WalletConfig> = {
     conflicts: ["mnemonic", "deterministic"]
   },
   mnemonic: {
-    normalize,
+    // if the mnemonic is empty, we want to throw an error because it is likely
+    // unexpected behavior
+    normalize: (mnemonic, config) => {
+      // if the mnemonic is `""` use `default` mnemonic generation.
+      if (mnemonic === "") {
```

Comment:
I'm not so sure this is even a good idea.

We probably should _validate_ mnemonics... and `""` is actually an invalid mnemonic.

Maybe we should close this PR.

Reply:
the library we use exposes the function `validateMnemonic()`, so I vote that we do that (either in this PR, or a separate one), and fail if an empty string is provided (unless there's some interface where it would be difficult to _not_ provide an empty string 🤷)

Reply:
Yeah, `validateMnemonic` is quite the busy function. Reading it at a glance it looks like it'd be costly, but even if it is most users will only need to call it once per start up, so it's probably fine?

I think it should be in another PR, so I'll close this one. Issue created: https://github.com/trufflesuite/ganache/issues/4075